### DISCUSSION TITLE
[Question] Difference between Authentication DoS and Deauth/Disassiciation Mode?

### DISCUSSION THREAD

**Discussion by WaGi-Coding** [aircrack-ng/mdk4#96] [General]
I wonder what the difference between them are regarding the results and targets.

Is Auth DoS for APs only, while the other is working for APs and Stations?
The end result kinda is the same i guess? The clients get kicked and/or freeze?

Would be happy about any information about that, i am quite new to this and won't test around live before i don't even know the differences and causes.

Greetings

**Comment by Heyholiday067** [comment]
Send deauth packet to victim is just like 

`A fake police block the road and say you cant walk on it`

Thus,you are impossibe to use the road!


But DOS is like
`You hire a lot of drivers to jam the road,so it is hard for you to walk.`

However in this case,if the road is wide enough,you can still walk,and thus attacker need to hire more and more car to jam it!
